pkg/distributor/distributor.go

```diff
@@ -324,8 +328,9 @@ func New(cfg Config, clientConfig ingester_client.Config, limits *validation.Ove
 			Help:      "The configured replication factor.",
 		}),
 		latestSeenSampleTimestampPerUser: promauto.With(reg).NewGaugeVec(prometheus.GaugeOpts{
-			Name: "cortex_distributor_latest_seen_sample_timestamp_seconds",
-			Help: "Unix timestamp of latest received sample per user.",
+			Namespace: "cortex",
```

Comment:
This is unnecessary change in this PR, and imho also a change in wrong direction. It makes finding metrics more difficult (instead of searching for full metric name like `cortex_distributor_latest_seen_sample_timestamp_seconds`, one needs to remove prefix and try again). I'd suggest 1) not do it here, and 2) in other PR remove all usage of `Namespace` field altogether.

Reply:
Fiiiiiine